cmd/tofu/commands.go

```diff
@@ -122,233 +142,307 @@ func initCommands(
 
 	commands = map[string]cli.CommandFactory{
 		"apply": func() (cli.Command, error) {
-			return &command.ApplyCommand{
-				Meta: meta,
+			return &pedanticWrapper{
+				PedanticCommand: &command.ApplyCommand{
+					Meta: meta,
+				},
 			}, nil
 		},
 
 		"console": func() (cli.Command, error) {
-			return &command.ConsoleCommand{
-				Meta: meta,
+			return &pedanticWrapper{
+				PedanticCommand: &command.ConsoleCommand{
```

Comment:
What are your thoughts on something like:
 https://gist.github.com/cam72cam/11b16e6ba97f714e5f42b02664e23171 to reduce the copy/paste changes to commands.go?

Reply:
cool, this is a really nice generic way to wrap the commands instead of having an inline per-command wrapping, I'll update the code with this approach.

Reply:
This has been updated.